tests/rustdoc/anchor-id-duplicate-method-name-25001.rs

```diff
@@ -27,20 +27,20 @@ impl<T> Bar for Foo<T> {
     //@ has - '//*[@id="associatedtype.Item"]//h4[@class="code-header"]' 'type Item = T'
     type Item=T;
 
-    //@ has - '//*[@id="method.quux"]//h4[@class="code-header"]' 'fn quux(self)'
-    fn quux(self) {}
+    //@ has - '//*[@id="method.quux"]//h4[@class="code-header"]' 'fn quux(self) -> T'
+    fn quux(self) -> T {}
```

Comment:
Also, while rustdoc supports it by virtue of not really resolving and typeck'ing function bodies, it'd rather see `loop {}` or `unimplemented!()` in there rather than a "dormant" type error.

Reply:
Oops, didn't even notice this won't compile.